File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/chat_message_list.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ChatMessageList:
    """OpenAI标准聊天消息列表"""
    object: str = None
    data: List[ChatMessage] = None
    has_more: bool = None
    first_id: str = None
    last_id: str = None

File: sdkwork-ai-sdk-go/types/chat_message_list.go
package types

// OpenAI标准聊天消息列表
type ChatMessageList struct {
	Object string `json:"object"`
	Data []ChatMessage `json:"data"`
	HasMore bool `json:"has_more"`
	FirstId string `json:"first_id"`
	LastId string `json:"last_id"`
}
